test/e2e/util/util.go

```diff
@@ -308,6 +308,29 @@ func EnsureNoPodsWithTooHighPriority(t *testing.T, ctx context.Context, client c
 	})
 }
 
+func EnsureNoContainersWithImagePullPolicyAlways(t *testing.T, ctx context.Context, client crclient.Client, hostedCluster *hyperv1.HostedCluster) {
```

Comment:
This is named `EnsureNoContainersWithImagePullPolicyAlways` but is actually doing `EnsureAllContainersHavePullIfNotPresentPolicy`